android/data/src/main/java/org/fossasia/openevent/data/Feedback.java

```diff
@@ -0,0 +1,37 @@
+package org.fossasia.openevent.data;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.github.jasminb.jsonapi.LongIdHandler;
+import com.github.jasminb.jsonapi.annotations.Id;
+import com.github.jasminb.jsonapi.annotations.Relationship;
+import com.github.jasminb.jsonapi.annotations.Type;
+
+import org.fossasia.openevent.core.auth.model.User;
+
+import io.realm.RealmObject;
+import io.realm.annotations.PrimaryKey;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+@Data
+@Builder
+@Type("feedback")
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString(exclude = "event")
+@JsonNaming(PropertyNamingStrategy.KebabCaseStrategy.class)
+@EqualsAndHashCode()
+public class Feedback extends RealmObject {
+
+    @Id(LongIdHandler.class)
+    @PrimaryKey
+    public Long id;
+    @Relationship("user")
+    public User user;
+    public Long rating;
```

Comment:
Rating should be float AFAIK

Reply:
I am not sure! It will make it float though if you say so

Reply:
Please confirm in API docs

Reply:
In the docs it says string
<img width="634" alt="screen shot 2018-05-10 at 11 45 49 am" src="https://user-images.githubusercontent.com/16715499/39855003-c24fb54e-5447-11e8-8f4f-9ad39b80e5e6.png">

Reply:
So it should be String, also make an issue in Server to ensure this is the desired behaviour as it should be float in my opinion

Reply:
sure